shared/featureconfig/flags.go

```diff
@@ -85,6 +85,11 @@ var (
 			"initial sync and disk-IO is one of the biggest bottleneck. This still saves finalized state in DB " +
 			"and start syncing from there",
 	}
+	connectToSlasherFlag = cli.BoolFlag{
```

Comment:
Please provide more context, ie. how is the slasher connected with the beacon node?

Reply:
added context